modules/ROOT/pages/troubleshoot-sso-errors.adoc

```diff
@@ -37,12 +37,12 @@ To troubleshoot this issue, publish the portal from the Builder.
 
 After logging in to the portal successfully, you cannot see the APIs in the API Carousel.
 
-There can be a a few issues that can cause this error to occur:
+There can be a few issues that can cause this error to occur:
 
 * The external identity provider group mapping is missing from Access Management.
 * The *Auth provider Default Scopes* field is not configured properly.
 
-=== External Identity Provider Group Mapping is Missing or Misconfigured
+=== External Identity Provider Group Mapping Is Missing or Misconfigured
```

Comment:
Why are you changing these titles? 
Can you run them through https://capitalizemytitle.com/